code/modules/mob/living/carbon/human/species/station/blank_vr.dm

```diff
@@ -8,6 +8,7 @@
 	var/eat_minerals = FALSE //HEAVY METAL DIET
 	var/base_species = null // Unused outside of a few species
 	var/selects_bodytype = FALSE // Allows the species to choose from body types intead of being forced to be just one.
+	var/list/traits = list() //Contains select neutral traits only unless you're a custom species.
```

Comment:
Uhhhh species other than custom species are global singletons. Every 'Tajaran' for example is a reference to a single global instance of `/datum/species/tajaran`. You can't just add traits to them, *everyone* will have those traits that is that species in the round. Eventually every Tajaran (or whatever) will have every trait suddenly added to them and This Is Bad.

Reply:
😮

Reply:
For further reference see human.dm:1152, in set_species(), where it sets `species` to the singleton in the `GLOB.all_species` list

Reply:
Spot on! I looked that and git grepped for everywhere else species is set

I'll need to go over it with a fine toothed comb as the last time I saw species anti-globalized it was really involved